app/Http/Controllers/SettingsController.php

```diff
@@ -283,6 +284,11 @@ public function support($id = null, $campnums = null)
 
                 }
             }
+            /** By Reena Nalwa #974 */
```

Comment:
@nalwa Can you please remove the name from the comments? We can always go back to history (git) and restore the file.

Reply:
@akuks I add name so that one can know who have done the code and for any discussion can directly connect to the that DEV.  without checking any git/commit history If there is issue with it which i am not aware of please let me know.